eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -566,7 +566,10 @@ class EclairImpl(appKit: Kit) extends Eclair with Logging {
           randomKey(),
           randomKey(),
           intermediateNodes.map(OnionMessages.IntermediateNode(_)),
-          destination match { case Left(key) => OnionMessages.Recipient(key, None) case Right(route) => OnionMessages.BlindedPath(route) },
```

Comment:
I initially modified this part a long time ago in the first versions of payment recipients, but that code has disappeared since then, so I'll revert that.

Reply:
Reverted in https://github.com/ACINQ/eclair/pull/2482/commits/855ce3b5ab0e54a70549a8bc62d1c941386ab2b1